README.md

```diff
@@ -1,6 +1,6 @@
-# OpenHAB Frigate SVR Binding
+# OpenHAB Frigate NVR Binding
 
-This is a comprehensive binding for the Frigate SVR system (https://docs.frigate.video/). It allows access to all configured
+This is a comprehensive binding for the Frigate NVR system (https://docs.frigate.video/). It allows access to all configured
```

Comment:
Frigate refers to itself as an NVR.
Your binding is FrigateSvr I know but I think this makes sense